### PR TITLE
Implement Joystick IsGamepad property

### PR DESCRIPTION
**Pull request by cra0zy**
http://community.monogame.net/t/how-to-detect-if-a-joystick-is-a-gamepad/8914

### CODE REVIEW

**Comment by tomspilman** [comment]
@cra0zy - This PR is conflicted now.

**Comment by cra0zy** [comment]
Fixed.

File: MonoGame.Framework/Input/JoystickCapabilities.cs [hunk 1]
```diff
@@ -22,6 +22,12 @@ public struct JoystickCapabilities
         public string Identifier { get; internal set; }
 
         /// <summary>
+        /// Gets or sets a value indicating whether the joystick is a gamepad.
```
Comment (tomspilman) [inline MonoGame.Framework/Input/JoystickCapabilities.cs]: The docs here are wrong.  The user cannot "set" this property... it is internal.

Comment (cra0zy) [inline MonoGame.Framework/Input/JoystickCapabilities.cs]: Fixed.

**Comment by tomspilman** [comment]
Thanks!

**Pull request merged**